README.md

```
@@ -67,27 +67,15 @@ $ tuist edit
 ```
 
 # Framework
+### Architecture
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)